From 269b3b84ed522db9bf9f2404bd0c29a6189e772f Mon Sep 17 00:00:00 2001
From: Bram Oosterhuis <dev@bybram.com>
Date: Fri, 7 Aug 2026 12:33:18 +0000
Subject: [PATCH 3/5] 0003-Callsign-not-generated-Json-Generator.patch

Reason for Change: This patch resolves a issue where  the callsign was not generated by JSONgenerator for configured plugins
Upstream-Status: Comcast changes
Signed-off-by: Karunakaran Amirthalingam <karunakaran_amirthalingam@cable.comcast.com>
---
 JsonGenerator/source/documentation_generator.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/JsonGenerator/source/documentation_generator.py b/JsonGenerator/source/documentation_generator.py
index 03b9f9d..f6d4c5e 100644
--- a/JsonGenerator/source/documentation_generator.py
+++ b/JsonGenerator/source/documentation_generator.py
@@ -1267,7 +1267,7 @@ def Create(log, args, schema, path, indent_size = 4):
                         method = method[:method.find('#')] + method[method.find(':', method.find('#')):]
 
                     if props and method not in skip_list and "#" not in method:
-                        to_skip = MethodDump(method, props, ("<callsign>" if document_type == "interface" else plugin_class), api, section_name, header, event, prop)
+                        to_skip = MethodDump(method, props, plugin_class, api, section_name, header, event, prop)
 
                         if to_skip:
                             skip_list.append(to_skip)
